### PR TITLE
Avoid applying partial dartfixes on CI

### PR DESCRIPTION
**Pull request by LongCatIsLooong**
Unblocks https://github.com/flutter/tests/pull/345

Add `bulkApply: false` to partial fixes so they don't automatically get applied on registered tests.

## Pre-launch Checklist

- [ ] I read the [Contributor Guide] and followed the process outlined there for submitting PRs.
- [ ] I read the [Tree Hygiene] wiki page, which explains my responsibilities.
- [ ] I read and followed the [Flutter Style Guide], including [Features we expect every widget to implement].
- [ ] I signed the [CLA].
- [ ] I listed at least one issue that this PR fixes in the description above.
- [ ] I updated/added relevant documentation (doc comments with `///`).
- [ ] I added new tests to check the change I am making, or this PR is [test-exempt].
- [ ] All existing and new tests are passing.

If you need help, consider asking for advice on the #hackers-new channel on [Discord].

[Contributor Guide]: https://github.com/flutter/flutter/wiki/Tree-hygiene#overview
[Tree Hygiene]: https://github.com/flutter/flutter/wiki/Tree-hygiene
[test-exempt]: https://github.com/flutter/flutter/wiki/Tree-hygiene#tests
[Flutter Style Guide]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo
[Features we expect every widget to implement]: https://github.com/flutter/flutter/wiki/Style-guide-for-Flutter-repo#features-we-expect-every-widget-to-implement
[CLA]: https://cla.developers.google.com/
[flutter/tests]: https://github.com/flutter/tests
[breaking change policy]: https://github.com/flutter/flutter/wiki/Tree-hygiene#handling-breaking-changes
[Discord]: https://github.com/flutter/flutter/wiki/Chat

### CODE REVIEW

**Comment by LongCatIsLooong** [comment]
@Piinks it looks like `bulkApply: false` also prevents the transform from running in tests. Are you ok with commenting related tests out for now?

File: packages/flutter/lib/fix_data/fix_material/fix_material.yaml [hunk 1]
```diff
@@ -865,6 +865,7 @@ transforms:
   # Changes made in https://github.com/flutter/flutter/pull/111706
   - title: "Migrate to 'trackVisibility'"
     date: 2022-09-15
+    bulkApply: false
```
Comment (Piinks) [inline packages/flutter/lib/fix_data/fix_material/fix_material.yaml:868]: I don't think these are the ones blocking, it's showTrackOnHover isn't it?

**Review by Piinks — COMMENTED** [review]
I am more concerned that the customer is using deprecated API we are about to remove in the next month. Can we send them a PR to get them off using the old API? We'll be doing that anyways in a few weeks.

**Comment by Piinks** [comment]
In general, I think using `bulkApply: false` is perfect for partial fixes. This case just seems unnecessary since we are literally about to delete that API.

File: packages/flutter/lib/fix_data/fix_material/fix_material.yaml (hunk 1 above)
Comment (Piinks) [reply packages/flutter/lib/fix_data/fix_material/fix_material.yaml:868]: Oh wait I misread, this is the right set of fixes. :)

**Review by Piinks — APPROVED** [review]
LGTM

**Pull request merged**